apps/client/components/Tooltip.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import { css } from "@styled-system/css";
+import type { PropsWithChildren, ReactNode } from "react";
+import React, { useState } from "react";
+
+interface TooltipProps extends PropsWithChildren {
+  content: ReactNode;
+}
+
+const Tooltip = ({ content, children }: TooltipProps) => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  const handleMouseEnter = () => {
+    setIsVisible(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsVisible(false);
+  };
```

Comment:
p4;
`useModalState` 훅을 사용할 수 있을 것 같아요. 모달에 제한되어 사용되는 훅이 아니니 네이밍을 바꾸는게 좋을 것 같네요! `useOpen`..?

```suggestion
const { isOpen, openModal, closeModal } = useModalState();
```

Reply:
useOpenState 로 수정했습니다~!